xml/cl.xml

```diff
@@ -5657,6 +5666,15 @@ server's OpenCL/api-docs repository.
                 <enum name="CL_DEVICE_NODE_MASK_KHR"/>
             </require>
         </extension>
+        <extension name="cl_intel_create_buffer_with_properties" supported="opencl">
+            <require>
+                <command name="clCreateBufferWithPropertiesINTEL"/>
+            </require>
+        </extension>
+        <extension name="cl_intel_mem_channel_property" supported="opencl">
+            <require comment="cl_mem_properties_intel">
+                <enum name="CL_MEM_CHANNEL_INTEL"/>
```

Comment:
```suggestion
                <enum name="CL_MEM_CHANNEL_INTEL"/>
            </require>
```

Looks like the merge went a little wrong.

Reply:
Oh, sorry, thank you for noticing!